Spigot-API-Patches/0002-Add-FastUtil-to-Bukkit.patch

```diff
@@ -16,7 +16,7 @@ index ed19c62e5a2dbc7e660eae61aef21295afdb8aaf..58fc279186e01a4703102227f387e962
 +        <dependency>
 +            <groupId>it.unimi.dsi</groupId>
 +            <artifactId>fastutil</artifactId>
-+            <version>8.2.2</version>
++            <version>8.5.2</version>
```

Comment:
I thought that the fastutil version had to be kept in sync with vanilla? correct me if I'm wrong.

Reply:
So long as there is no ABI breaks, should be fine, this is API after all and any breaking updates would ideally come from mojang or maybe at a "release"